src/hmrc-content-guide/notices/index.njk

```diff
@@ -10,22 +10,20 @@ layout: twoColumnPage.njk
 
 <h2 class="govuk-heading-l">When to use</h2>
 
-<p class="govuk-body">Use this pattern to design an excise, Soft Drinks Industry Levy or VAT notice that includes force of law - for examples see:</p>
+<p class="govuk-body">
+Use this pattern to design an excise, Soft Drinks Industry Levy or VAT notice that includes <a href="https://design.tax.service.gov.uk/hmrc-content-guide/force-of-law" class="govuk-link">force
```

Comment:
strip out 'https://design.tax.service.gov.uk' so the link works across our environments

Reply:
I am not sure what it's supposed to look like instead?

Reply:
href="/hmrc-content-guide/force-of-law/"